feat(curriculum): forum leaderboard project (#50241)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/644f5855e8e462024c1c4501.md

---
id: 644f5855e8e462024c1c4501
title: Step 75
challengeType: 0
dashedName: step-75
---

# --description--

Last step! you need to add the argument named `id` inside the template literal after putting a slash `/` at the end of the template literal.

# --hints--

You should put a slash `/` after the `selectedCategory.className`

```js
const regex = /const\s+url\s*=\s*`\$\{FORUM_CATEGORY_URL\}\$\{selectedCategory.className\}\/.*`/g;
assert(code.match(regex));
```

> You should add argument `id` inside the template literal, after the slash `/`.

Reviewer: Should the colution be:
```
    const  url = `${FORUM_CATEGORY_URL}${selectedCategory.className}/${id}`;
```

OR

```
    const  url = `${FORUM_CATEGORY_URL}${selectedCategory.className}/${selectedCategory.id}`;
```

Author: const url = `${FORUM_CATEGORY_URL}${selectedCategory.className}/${id}`;

---
id: 644f5855e8e462024c1c4501
title: Step 55
challengeType: 0
dashedName: step-55
---

# --description--

Last step! you need to add the argument named `id` inside the template literal after putting a slash `/` at the end of the template literal.

# --hints--

You should put a slash `/` after the `selectedCategory.className`

```js
const regex = /const\s+url\s*=\s*`\$\{FORUM_CATEGORY_URL\}\$\{selectedCategory.className\}\/.*`/g;
assert(code.match(regex));
```

> You should add argument `id` inside the template literal, after the slash `/`.

Reviewer 2: ```suggestion
You should add an embedded expression for the `id` after the forward slash `/`.
```
